apps/docs/docs/troubleshoot.md

```diff
@@ -0,0 +1,22 @@
+# Troubleshoot 
+##  Why is my Typescript intellisense *slow*? 
+
+You'll need to enable `strictNullChecks` in your `tsconfig.json`:
```

Comment:
I think it would help here to explain that despite `strict` being enabled it might be possible that `strictNullChecks` might be disabled explicitly, so it'll need to be set to `true` to ensure that it doesn't cause issues with the zod typings.

Reply:
clarified!